Week1/homework/app.js

```diff
@@ -1,11 +1,145 @@
 'use strict';
 
 {
+// **1.1** declaring an array
+
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+    `The_Goldfinch`,
+    `A_Garden_of_Earthly_Delights`,
+    `Beloved`,
+    `There_Once_Lived_a_Woman_Who_Tried_to_Kill_Her_Neighbors_Baby`,
+    `The_House_of_the_Spirits`,
+    `The_Left_Hand_of_Darkness`,
+    `The_Unwomanly_Face_of_War`,
+    `Chernobyl_Prayer`,
+    `Runaway`,
+    `The_Year_of_Magical_Thinking`
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
-}
+  /* **1.3** function that generate a `ul` with `li` elements for each book title
+  
+  function createListOfBookTitles() {
+
+    const emptyList = document.createElement("ul");
+
+    function placeList() {
+      const placeForList = document.body;
+      placeForList.insertBefore(emptyList, placeForList.childNodes[2]);
+    };
+    placeList();
+
+    function addTitlesToList() {
+      let x;
+      for (x in bookTitles) {
+        const newListItem = document.createElement("li");
+        const titleItem = document.createTextNode(bookTitles[x]);
+        newListItem.appendChild(titleItem);
+        emptyList.appendChild(newListItem);
+      };
+    };
+    addTitlesToList();
+  };
+  createListOfBookTitles();
+  */
+
+// **1.4** Making an object
+
+  const Books = bookTitles.reduce((acc, elem) => {
+    acc[elem] = {title:``, author:``, original_language:``};
+    return acc;
+  }, {});
+
+// Adding content
+
+  Books.The_Goldfinch.title = `The Goldfinch`;
```

Comment:
Instead of this method use an object literal to define the books object.
Example of an Object literal:
```
const fruitObject = {
  banana: {
    color: "yellow",
    weight: 200,
  },
  orange: {
    color: "orange",
    weight: 150,
  },
  apple: {
    color: "red",
    weight: 120
  },
  lime: {
    color: "green",
    weight: 60
  }
};
```

Reply:
done